package.json

```diff
@@ -18,7 +18,14 @@
   },
   "homepage": "https://github.com/strophe/strophejs-plugin-stream-management#readme",
   "devDependencies": {
-    "rollup": "^0.45.2"
+    "http-server": "^0.10.0",
```

Comment:
Are all these new deps being used?

Reply:
almost all, I was having issues in making chrome headless running the tests, I guess I will remove all not used ones now and  re-add them back

Reply:
just checked it now and they are all used